build/chip/java/rules.gni

```diff
@@ -194,6 +349,24 @@ template("android_library") {
   }
 }
 
+template("android_binary") {
```

Comment:
It's not really an android binary, but rather, a java binary right?

Reply:
Yes, just try to align the name with android_library whose output is a Java library.